src/components/StaticContent/HomepageLeaderBoard.re

```diff
@@ -14,6 +17,17 @@ let make = () => {
       <Box width=[|1., 1., 0.59|] className=Styles.infoCardContainer>
         <Card className=Styles.infoCardStyles>
           <MonthlyContribution numberOfLeaders=6 />
+          <br />
+          <Rimble.Box width=[|1.|]>
+            <Button
+              className=Styles.centerItemsMargin
+              onClick={e => {
+                ReactEvent.Form.preventDefault(e);
+                clearAndPush({j|/#leaderboards/monthly-contribution|j});
+              }}>
+              "View Other Leaderboards - "->restr
```

Comment:
We can remove the trailing dash? "View other Leaderboards"

Reply:
Fixed!!